win32ss/user/ntuser/defwnd.c

```diff
@@ -619,14 +619,6 @@ IntDefWindowProc(
          if (!Wnd->spwndOwner) break;
```

Comment:
For the NTUSER changes, I would actually suggest you create a separate PR just for these.